src/style.css

```diff
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.form {
+  display: flex;
+  margin: 20px auto;
+}
+
+.form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.form_small {
+  width: 100%;
+  height: 42px;
+}
+
+.form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.form__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  color: #3d4e61;
+  outline: none;
+}
+
+.form__input_big {
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.form__input_small {
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+.form__img {
+  position: absolute;
+}
+
+.form__img_big {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.form__img_small {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.form__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form__input:focus {
```

Comment:
on focus input text should become bolder
<img width="257" alt="image" src="https://user-images.githubusercontent.com/66749919/206840917-12413365-d6fb-46a0-9984-22bbe6477d72.png">